vars/prepareDefaultValues.groovy

```diff
@@ -6,24 +6,22 @@ import groovy.transform.Field
 @Field STEP_NAME = getClass().getName()
 
 void call(Map parameters = [:]) {
-    handlePipelineStepErrors (stepName: 'prepareDefaultValues', stepParameters: parameters, echoDetails: false) {
```

Comment:
out of curiosity: is there any particular reason for removing `handlePipelineStepErrors` from here?
I guess it is a chicken-or-egg-problem since we cannot use the configuration framework inside the handle step error when this call here is present.

In case this is the reason I consider this to be a code-smell. Would it be possible to get ride of that method in favor of some utils in `src`?

Reply:
I agree we should have a look at this. Shall I add a `//TODO`?

Reply:
I think it's OK to not use `handlePipelineStepErrors` here. We do the same in other utility-like steps ...

Reply:
I'm not fighting against removing the handlePipelineStepError from here. My point is: I would like to outline a code smell, which we might want to fix (of course not in this PR).